pocs/state/machine.py

```diff
@@ -179,7 +179,7 @@ def goto_next_state(self):
 
         self.logger.debug("Transition method: {}".format(call_method))
 
-        caller = getattr(self, call_method, 'park')
+        caller = getattr(self, call_method, self.park)
```

Comment:
Could you change the name of call_method to transition_method_name and caller to transition_method? That would help me when coming in here and wondering what is going on.